tests/models/audio_spectrogram_transformer/test_feature_extraction_audio_spectrogram_transformer.py

```diff
@@ -136,6 +142,50 @@ def test_call(self):
         for enc_seq_1, enc_seq_2 in zip(encoded_sequences_1, encoded_sequences_2):
             self.assertTrue(np.allclose(enc_seq_1, enc_seq_2, atol=1e-3))
 
+    def test_call_with_timem_and_freqm(self):
```

Comment:
Nice tests! Let's also make sure this also works when passing only `timem` or `freqm`

Reply:
And a test to make sure we raise a warning if torchaudio is not installed and we use masking